src/python/components/Graphistry.py

```diff
@@ -56,15 +56,21 @@ def plot(self, g):
 
     def test_login(self, verbose=True):
         try:
-            graphistry.register()
+            graphistry.register(**self.cfg)
+            if not (("username" in self.cfg) and ("password" in self.cfg)):
```

Comment:
they might use an api token or other reg method, so really should check that the jwt token got set: https://github.com/graphistry/pygraphistry/blob/ac9fb07493cc8d5cacf8a7c41cfdf0de565a1d87/graphistry/pygraphistry.py#L474